<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>9号館POS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>781</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="regi">
     <widget class="QLabel" name="total_amount">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>361</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>10000</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buy">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>560</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>購入</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="scanned_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>801</width>
        <height>591</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="autoScrollMargin">
       <number>8</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::TargetMoveAction</enum>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QPushButton" name="undo">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>160</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Takao Pゴシック</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1つ戻す</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clean">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>360</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Takao Pゴシック</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>すべて削除</string>
      </property>
     </widget>
     <widget class="QLabel" name="scan_amount">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>60</y>
        <width>731</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>ノムさんの*  100</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="payment">
     <widget class="QLabel" name="msg_amountis">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>お支払い金額は</string>
      </property>
     </widget>
     <widget class="QLabel" name="purchase">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>350</y>
        <width>531</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="msg_touch">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>580</y>
        <width>701</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>カードをタッチしてください</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pay_cash">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>60</y>
        <width>511</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>現金で支払う</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="back_pos">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>410</y>
        <width>511</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>商品登録に戻る</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="thankyou">
     <widget class="QLabel" name="msg_arigato">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>1051</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>お買い上げありがとうございます</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="msg_chargeis">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>チャージ残高は</string>
      </property>
     </widget>
     <widget class="QLabel" name="charge_balance">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>440</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>０円</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="msg_back">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>650</y>
        <width>1051</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>数秒後に商品登録画面に戻ります...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="msg_who">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>1051</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ノムさん</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
